README.md

```diff
@@ -1,24 +1,203 @@
+    someone page through slides that present bits and pieces of the same
+    code.
+
+1.  It enables instructors to be more responsive to "what if?"
+    questions. Where a slide deck is like a railway track, live coding
+    allows instructors to go off road and follow their learners'
+    interests.
+
+1.  People learn more than we realize we are teaching by watching
+    *how* instructors do things.
+
+1.  It slows the instructor down: if she has to type in the program as
+    she goes along, she can only go twice as fast as her learners,
+    rather than ten times faster as she could with slides.
+
+1.  Learners get to see instructors' mistakes *and how to diagnose
+    and correct them*. Novices are going to spend most of their time
+    doing this, but it's left out of most textbooks.
+
+1.  Watching instructors make mistakes shows learners that it's all
+    right to make mistakes of their.  Most people model the behavior of
+    their teachers: if the instructor isn't embarrassed about making and
+    talking about mistakes, learners will be more comfortable doing so
+    too.
+
+Live coding doesn't make slides irrelevant.
+Instead, they serve two purposes:
+
+1.  Instructors read them *before* the lesson to be sure they have a
+    clear picture of the lesson's flow.
+
+1.  Learners read them *after* the lesson for review and reference.
+    They can also be given a printed copy of the slides to mark up
+    during the lesson so that they don't have to take notes from
+    scratch.
+
+The first point is essential for supporting collaboration among instructors.
+In order for two or more people to teach together effectively,
+they must have a shared road map of what they are going to do.
+They may improvise on top of that like jazz musicians improvise on top of a score,
+but a quick review of a shared slide deck before a class
+can prevent a lot of crossed wires, dropped balls, and other metaphors.
+
+These slides also help instructors share new and improved ideas with their peers.
+If one person comes up with a clever way to explain a complex idea,
+she can (and should) submit a pull request to add her discovery to the slides.
+Other people who are teaching this material can watch for incoming changes,
+or look at the differences between the slides as they were the last time they taught
+and the slides as they are today
+and see what we have collectively learned about how best to teach this topic.
+
+## How to View
+
+These slides use [Reveal.js][reveal].
+The first time you want to view them on your computer,
+you must:
+
+1.  Check out [this repository][repo] from GitHub.
+
+1.  Go into the project's `slides` directory.
+
+1.  Run `npm install` to install the JavaScript packages needed by [Reveal.js][reveal].
+
+You only need to do these steps once.
+After that,
+you can view the slides at any time:
+
+1.  Run `npm start` in the `./slides/` directory to run a local server.
```

Comment:
May want to mention that server is set to run on port 8081 and will error if that port is in use